app/controllers/propertyLinking/UploadController.scala

```diff
@@ -177,7 +177,7 @@ class UploadController @Inject() (
         )
     }
 
-  def continue: Action[AnyContent] =
```

Comment:
can you / have you ran the acceptance tests against this change? they fail for me every time against the property linking block of tests...but manually its all fine. If they run fine for you it could be a joeys machine issue